test/integration/auth/test_auth_api.py

```diff
@@ -17,10 +17,10 @@ class TestDssApi(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        cls.client = hca.auth.AuthClient(swagger_url="https://auth.dev.data.humancellatlas.org/swagger.json")
+        cls.client = dbio.auth.AuthClient(swagger_url="https://auth.ucsc.ucsc-cgp-redwood.org/swagger.json")
```

Comment:
imo. we should nuke the support for fusillade, in my experience the cli functionality does not really work. (but that could be operator problem)

Reply:
Will address in a later PR - see #22